frontend/packages/dev-console/src/components/import/pipeline/PipelineTemplate.tsx

```diff
@@ -55,7 +69,7 @@ const PipelineTemplate: React.FC = () => {
     return () => {
       ignore = true;
     };
-  }, [image.selected, isDockerStrategy, setFieldValue]);
+  }, [resources, image.selected, isDockerStrategy, setFieldValue, pipelineStorageRef]);
```

Comment:
I don't think you need `pipelineStorageRef` in the dependency list.

Reply:
Interesting... pretty sure it's a lint to have all things referenced in the dependency list - but I will try to remove it.

Reply:
Yup, seems not to be an issue - not sure really what the criteria is... will need to look up the linter rule.